packages/kubernetes-1.19/kubelet-config

```diff
@@ -93,6 +93,12 @@ topologyManagerPolicy: {{settings.kubernetes.topology-manager-policy}}
 {{#if settings.kubernetes.pod-pids-limit includeZero=true}}
 podPidsLimit: {{settings.kubernetes.pod-pids-limit}}
 {{/if}}
+{{#if settings.kubernetes.image-gc-high-threshold-percent includeZero=true}}
+imageGCHighThresholdPercent: {{default "85" settings.kubernetes.image-gc-high-threshold-percent}}
+{{/if}}
+{{#if settings.kubernetes.image-gc-low-threshold-percent includeZero=true}}
+imageGCLowThresholdPercent: {{default "80" settings.kubernetes.image-gc-low-threshold-percent}}
+{{/if}}
```

Comment:
We need to enforce the [validation rules](https://kubernetes.io/docs/reference/config-api/kubelet-config.v1beta1/) somehow:
>  imageGCHighThresholdPercent is the percent of disk usage after which image garbage collection is always run. The percent is calculated by dividing this field value by 100, so this field must be between 0 and 100, inclusive. When specified, the value must be greater than imageGCLowThresholdPercent. 

> imageGCLowThresholdPercent is the percent of disk usage before which image garbage collection is never run. Lowest disk usage to garbage collect to. The percent is calculated by dividing this field value by 100, so the field value must be between 0 and 100, inclusive. When specified, the value must be less than imageGCHighThresholdPercent. 

In the API, we should reject values outside of `0 <= n <= 100`, instead of accepting any `i32`.

Here in the template we need a way to ensure that we don't render an illegal configuration, which will mean comparing the values for the two settings.